File: testdata/bench/moving_avg.py
n = 200000
window = 10

data = []
for i in range(n):
    data.append((i * 13) % 1000)

sum_val = 0
for i in range(window):
    sum_val += data[i]

out = 0
for i in range(window, n):
    sum_val += data[i]
    sum_val -= data[i-window]
    out += sum_val // window

print(out)

File: testdata/bench/moving_avg.go
package main

import "fmt"

func main() {
	n := 200000
	window := 10

	data := make([]int, n)
	for i := 0; i < n; i++ {
		data[i] = (i * 13) % 1000
	}

	sum := 0
	for i := 0; i < window; i++ {
		sum += data[i]
	}

	out := 0
	for i := window; i < n; i++ {
		sum += data[i]
		sum -= data[i-window]
		out += sum / window
	}
	fmt.Println(out)
}
